src/components/Atoms/Checkbox/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import check from "../../../assets/svg/check.svg";
+
+const Checkbox = ({ onClick }) => {
+  return (
+    <StyledCheckbtn onClick={onClick}>
```

Comment:
```suggestion
    <StyledCheckbtn onClick={onClick} type="button">
```
htmlのbutton要素はformの中で使うのが念頭に置かれているので、formの中に置かれたうえでクリックすると送信処理をしてしまいます。
本課題ではformを使わないので修正の必要はありませんが、type="button"としておくとそのような挙動が防げるので、このような属性があるということだけ知っておいて損はないと思います！
https://developer.mozilla.org/ja/docs/Web/HTML/Element/button#attr-type

Reply:
ありがとうございます。